storage/etcd/standup.sh

```diff
@@ -0,0 +1,109 @@
+#!/bin/bash
+
+if [ "$EUID" -ne 0 ]
```

Comment:
Admittedly we've used these scripts less and less. I think instructions for downloading the binary releases would suffice https://github.com/coreos/etcd/releases